MediaAdminBundle/Resources/public/coffee/GalleryView.coffee

```diff
@@ -67,9 +67,9 @@ GalleryView = OrchestraView.extend(
 
   mediaSelect : (event) ->
     event.preventDefault()
-    mediaModalContainer = @$el.parents(".mediaModalContainer")
-    intputName = mediaModalContainer.data('input')
+    modalContainer = @$el.closest(".mediaModalContainer")
```

Comment:
Does it work well in nested modals ? For instance on a mediamodal inside a block form modal ?